### DISCUSSION TITLE
发现一个问题，执行安装命令后会锁上网卡文件，后面的自动配置命令就会失效

### DISCUSSION THREAD

**Discussion by xkatld** [oneclickvirt/oneclickvirt.github.io#9] [Ideas]
如题，需要执行chattr -i /etc/network/interfaces才能完成使用

**Comment by spiritLHLS** [comment]
PVE? 自动配置网关后默认是锁上的，没毛病

**Comment by spiritLHLS** [comment]
如果有问题 https://github.com/oneclickvirt/pve/issues 对应仓库发issues吧
